src/Collectors/ScheduledTaskCollector.php

```diff
@@ -66,10 +66,10 @@ protected function startTransaction(string $transaction_name): Transaction
         );
     }
 
-    protected function stopTransaction(string $transaction_name, int $result): void
+    protected function stopTransaction(string $transaction_name, ?int $result): void
     {
         // Stop the transaction and measure the time
-        $this->agent->stopTransaction($transaction_name, ['result' => $result]);
+        $this->agent->stopTransaction($transaction_name, ['result' => (int) $result]);
```

Comment:
If it's null, will it convert to `0`?

Reply:
yep, not sure what to do with it tbh. or when would it return null. Usually if it returns at all then it means the command worked, because nothing crashed, but I have no idea why it'd happen for @cykirsch .